src/sentry/workflow_engine/typings/notification_action.py

```diff
@@ -0,0 +1,113 @@
+from dataclasses import dataclass
+
+from sentry.notifications.models.notificationaction import ActionTarget
+from sentry.workflow_engine.models.action import Action
+
+"""
+Keys that are excluded from the action data blob.
+We don't want to save these keys because:
+- uuid: maps to action id
+- id: maps to action type
+"""
+EXCLUDED_ACTION_DATA_KEYS = ["uuid", "id"]
```

Comment:
currently not saving the uuid since we are going to replace with action.id, but i am also considering saving it in the data blob incase something bad happens and we need it to recover? 🤔